ios/browser/api/brave_wallet/brave_wallet_provider_delegate_ios+private.h

```diff
@@ -26,8 +26,14 @@ class BraveWalletProviderDelegateBridge
   url::Origin GetOrigin() const override;
   void RequestEthereumPermissions(
       RequestEthereumPermissionsCallback callback) override;
-  void GetAllowedAccounts(bool include_accounts_when_locked,
+  void GetAllowedAccounts(mojom::CoinType type,
+                          bool include_accounts_when_locked,
                           GetAllowedAccountsCallback callback) override;
+  void RequestSolanaPermission(
+      RequestSolanaPermissionCallback callback) override {}
+  void IsSelectedAccountAllowed(
+      mojom::CoinType type,
+      IsSelectedAccountAllowedCallback callback) override {}
```

Comment:
iOS will need bridged Obj-C counterpart methods in `BraveWalletProviderDelegate` for these new C++ methods

Reply:
fixed iin 09c8e944d6359048b35a8cb55113e07ed5b2163a